tests/unit/auth/oauth1/tokens.ts

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Opening up another PR to remove this line, it is simply a misconfigured eslint rule. A bit silly